src/pages/help.ts

```diff
@@ -96,7 +96,7 @@ export function getCategoryPage(
     );
 
   // Get current offset
-  let offset = parseInt(currentOffset);
+  let offset = parseInt(currentOffset ?? "0");
```

Comment:
This case is already handled by the NaN check.
Why we do a NaN check.. I can't remember and I'm too unbothered to find out, HA
```suggestion
  let offset = parseInt(currentOffset);
```

Reply:
When building, TypeScript complains sooo... it was just to satisfy the build

Reply:
Anything to satisfy the build 😩

Reply:
Fine for now though since I already committed upsi, rewrite bouta come it